docs/LINUX.md

```diff
@@ -74,6 +74,20 @@ or `flatpak install --user flathub com.github.marktext.marktext` to install for
 
 To run MarkText just execute `flatpak run com.github.marktext.marktext` or click on the MarkText icon in your application launcher.
 
+## Arch User Repository
```

Comment:
Please move the AUR section behind Flatpak because `### Update` is a part of Flatpak.